addons/knobs/src/html/registerKnobs.js

```diff
@@ -0,0 +1,60 @@
+import addons from '@storybook/addons';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { subscriptionsStore } from '@storybook/core/client';
```

Comment:
You can add it as devDependency, `subscriptionsStore` is a global singleton

But maybe a better approach would be to introduce an event for registering subscriptions

Reply:
I'll do the latter